awscli/customizations/wizard/app.py

```diff
@@ -104,6 +106,10 @@ def is_prompt_visible(self, value_name):
             return False
         return self._prompt_meets_condition(value_name)
 
+    def prompt_details_visible_by_default(self, value_name):
```

Comment:
For this method, let's add an `is` to better indicate it returns a boolean and since it does not use `has` so we do not have to worry about it reading weird.